doc/howto/instances_troubleshoot.md

```diff
@@ -59,7 +59,41 @@ To troubleshoot the problem, complete the following steps:
    If it is, and if you cannot figure out the source of the error from the log information, open a question in the [forum](https://discourse.ubuntu.com/c/lxd/).
    Make sure to include the log files you collected.
 
-## Troubleshooting example
+## Troubleshooting examples
+
+### Debug `systemd` `init`
+
+Here is how to ask `systemd` to enable [debug level messages](https://www.freedesktop.org/wiki/Software/systemd/Debugging/):
```

Comment:
I'm wondering if we can use a different name for the instance here (and in the example below :/); conflating `systemd` the container name with systemd the init system adds cognitive overhead for folks without a ton of linux experience.

Reply:
I agree - calling it `systemd-container` will make it clearer.

```suggestion
Here is how to configure the `systemd-container` to enable [debug level messages](https://www.freedesktop.org/wiki/Software/systemd/Debugging/):
```

Reply:
Thank you both, it was indeed confusing. I've fixed the existing example too `s/systemd/rhel7/` as it's specifically called out that it's a RHEL 7 container.